scripts/setup_offline_updates.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+usage() {
+    echo
+    echo "Usage: $0 <repo-type> <tar-file>"
+    echo
+    echo "Creates or updates an XCP-ng offline <repo-type> repository from the <tar-file> archive."
+    echo
+    echo "Arguments:"
+    echo "  repo-type   The repository type, based on its name on online mirrors. Example : 'updates'."
+    echo "  tar-file    The path to the archive containing the repository files."
+}
+
+if [ -z "$2" ]; then
+    echo "Error: argument(s) missing."
+    usage
+    exit 1
+fi
+
+set -eu
+
+REPO_TYPE="$1"
+TAR_FILE="$2"
+REPOS_DIR="/var/local/xcp-ng-repos"
+REPONAME="xcp-ng-offline-$REPO_TYPE"
+
+if [ ! -e "$TAR_FILE" ]; then
+    echo "Error: file $TAR_FILE not found."
+    exit 2
+fi
+
+if ! tar -tf "$TAR_FILE" | grep -q "^${REPO_TYPE}/$"; then
+    echo "Error: directory $REPO_TYPE/ not found in archive $TAR_FILE."
+    exit 3
+fi
+
+mkdir -p "$REPOS_DIR"
+
+if [ -d "$REPOS_DIR/$REPO_TYPE" ]; then
+    echo "- Deleting old repo at $REPOS_DIR/$REPO_TYPE"
+    rm "$REPOS_DIR/$REPO_TYPE" -r
```

Comment:
Maybe add `-f` to be sure, possibly subject to a `-f` flag on this script?

Reply:
I'd rather have the script fail if `rm -r` fails for some reason, to have an occasion to understand why it happened.

Reply:
That's why I was suggesting to let the user specify `-f` if needed

Reply:
Maybe for AirGapped v2 (the next iteration). If I need to add non positional arguments, I'll switch to python with argparse :)